templates/Dockerfile-rpm.tmpl

```diff
@@ -93,6 +93,20 @@ RUN { \
     } \
     && yum clean all
 
+# install cmake, devtoolset and its dependencies to build azure sdk
+RUN yum -y install  perl-IPC-Cmd libuuid-devel cmake3
```

Comment:
It is a good idea to run `yum clean all` to ensure we have a small docker image, and small docker cache usage for intermediate steps.